### DISCUSSION TITLE
Help with cutom HA Entitys

### DISCUSSION THREAD

**Discussion by OlderHolger1966** [zachowj/hass-node-red#244] [Q&A]
Hello, 
ich try to create an Helper entity for myHA Automation. 
I get the Error Message : 
`This node requires [Node-RED custom integration version 1.0.4+ ](https://github.com/zachowj/hass-node-red) to be installed in Home Assistant for it to function.`
But the newest version is 0.62.3

I alredy tryed to move the custom _integrations folder to my HA but it didnt work. 

Node red and HA are running in seperate Docker conainer on Version "3.1.3" and "2024.1.1". And all other Nodes are completly working.

**Comment by zachowj** [comment]
Check the [diagnostics popup](https://zachowj.github.io/node-red-contrib-home-assistant-websocket/guide/diagnostics.html), if you don't see the version number you need next to `Companion version`. You don't have the [custom component](https://github.com/zachowj/hass-node-red) installed correctly.
